core/common/src/rpc/core/RpcInvocation.ts

```diff
@@ -118,10 +119,8 @@ export class RpcInvocation {
   }
 
   private async resolve(): Promise<any> {
+    const currentRequest = await RpcConfiguration.requestContext.deserialize(this.request);
```

Comment:
I would feel better if this was inside the try block.

Reply:
it can't be inside the try block since it is referenced in the catch. If you're worried about it failing I guess we'd need another try, but what would we do other than throwing?

Reply:
Couldn’t it be declared outside of the try?

Reply:
I’ve seen it fail in our apps.

Reply:
done. I made a new (public) static method so I can write a test that the log message is correct.

Reply:
Looks good.